sources/packages/web/src/services/InstitutionService.ts

```diff
@@ -52,7 +53,7 @@ export class InstitutionService {
   }
 
   async updateInstitution(
-    data: InstitutionContactAPIInDTO,
```

Comment:
Optional. We can add comments to the method.

Reply:
Comments added.